server/charts/openscout/values.yaml

```diff
@@ -52,14 +52,10 @@ images:
     repository: docker.elastic.co/logstash/logstash-oss
     # Tag is elkversion
 
-# For each volume mount
-volumes:
-  pipeline:
-    hostPath: /opt/openscout/elk/pipeline
-    mountPath: /usr/share/logstash/pipeline
-  objectengine:
-    hostPath: /opt/openscout/openscout-vol
-    mountPath: /openscout/server/images    
+# For each model
```

Comment:
the model.url in values.yaml should be commented so that we can fall back on models shipped as part of the Docker image.